### PR TITLE
a bug on softamx and K.set_image_data_format("channels_first")

### PR DESCRIPTION
**Pull request by duducheng**
Hi Zhou, 

We have been trying your amazing code for days, thanks for your work!

We found a bug on your README on how to use your pretrained model for segmentation.

Your model code define a channel-first implementation
```python
K.set_image_data_format("channels_first")
```
, where inputs: `B x C x D x H x W`. Thereby, your output `final_convolution` is also `B x C x D x H x W`.

If you run.
```python
output = Activation('softmax')(final_convolution)
```
The default `softmax` is compute the normalized term on the `axis=-1`, i.e. the `W`-axis, which is usually not what people want.

### CODE REVIEW

**Comment by MrGiovanni** [comment]
Hi Jiancheng,

Thank you for your helpful comments. I have now updated the README file following your suggestion.

I did not use 'softmax' and 'multi-class segmentation' in the paper, that is, 
`num_class = 1, activation = 'sigmoid'`
considering the foreground as positives and background as negatives. It predicts segmentation masks properly in that case. 

But as you mentioned, the bug does exist when activating using 'softmax'. Many thanks.

Best regards,
Zongwei
